extramodules/it/eng/jhove/module/png/PngModule.java

```diff
@@ -379,7 +378,7 @@ public final int parse(final InputStream inputStream,
 										   IdentifierType.URL));
 		_specification.add (doc);
 
-		Signature sig = new InternalSignature ("PNG", SignatureType.MAGIC,
+		Signature sig = new InternalSignature (FORMAT[0], SignatureType.MAGIC,
```

Comment:
I'm just becoming aware that this approach does introduce an issue, although it's still an improvement. `FORMAT[0]` isn't as easy to read, but using a single source is a good idea. There's a couple of solutions come to mind but to establish consistency across the codebase I'll need to consider some other files and cases. This is fine for now as it weeds out the repetition.